modular_skyrat/modules/food_replicator/replicator_designs/replicator_medical.dm

```diff
@@ -0,0 +1,67 @@
+/datum/design/sutures
+	name = "Sutures"
+	id = "slavic_suture"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/stack/medical/suture
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/mesh
+	name = "Regenerative Mesh"
+	id = "slavic_mesh"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 150)
+	build_path = /obj/item/stack/medical/mesh
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/bruise_patch
+	name = "Bruise Patch"
+	id = "slavic_bruise"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 250)
+	build_path = /obj/item/reagent_containers/pill/patch/libital
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/burn_patch
+	name = "Burn Patch"
+	id = "slavic_burn"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 250)
+	build_path = /obj/item/reagent_containers/pill/patch/aiuri
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/gauze
+	name = "Medical Gauze"
+	id = "slavic_gauze"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 100)
+	build_path = /obj/item/stack/medical/gauze
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/epi_pill
+	name = "Epinephrine Pill"
+	id = "slavic_epi"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 75)
+	build_path = /obj/item/reagent_containers/pill/epinephrine
+	category = list(RND_CATEGORY_INITIAL, RND_CATEGORY_NRI_MEDICAL)
+
+/datum/design/conv_pill
+	name = "Convermol Pill"
+	id = "slavic_conv"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 75)
+	build_path = /obj/item/reagent_containers/pill/convermol
```

Comment:
That's not actually a typepath that exists in the code, just saying.

Reply:
It does - I've added it in the reagents folder.
![IMG_20230731_002211.jpg](https://github.com/Skyrat-SS13/Skyrat-tg/assets/42087567/f4ff1a92-3438-4e09-98aa-4964c2e24255)